BasePolicies/CONTRIBUTING.md

```diff
@@ -19,6 +19,7 @@ and the Node.js project.
 
 * A **Contributor** is any individual creating or commenting on an issue or pull request.
 * A **Committer** is a subset of contributors who have been given write access to the repository.
+* A **Collaborator** is a committer, but may use more time labelling and closing issues, rather than commit code.
```

Comment:
I could be wrong (someone please clarify), but IIRC "collaborator" isn't quite an official term, even though we use it frequently.

Perhaps it might be useful to add some wording similar to

```
* Informally, a **Collaborator** is...
```